config.cjs

```diff
@@ -0,0 +1,253 @@
+const YAML = require('yaml');
+const StyleDictionary = require('style-dictionary');
+const nunjucks = require('nunjucks');
+const path = require('path');
+const fs = require('fs');
+
+const env = nunjucks.configure(path.join(__dirname, 'docs'));
+
+env.addFilter('log', function(x) { console.log(x); });
+
+const { fileHeader, formattedVariables } = StyleDictionary.formatHelpers;
+
+/**
+ * Copy the repo license (MIT) into file headers
+ */
+StyleDictionary.registerFileHeader({
+  name: 'redhat/legal',
+  fileHeader(defaultMessage) {
+    return [
+      ...defaultMessage,
+      '',
+      '@license',
+      ...fs.readFileSync(path.join(__dirname, 'LICENSE'), 'utf8').split('\n'),
+    ];
+  },
+});
+
+/**
+ * Transform a record or box-shadow properties to a css box-shadow property value
+ * @see https://design-tokens.github.io/community-group/format/#shadow
+ */
+StyleDictionary.registerTransform({
+  type: 'value',
+  name: 'shadow/css',
+  matcher: token => token.type === 'shadow',
+  transformer: ({ value: { offsetX, offsetY, blur, spread, color }}) =>
+    `${offsetX} ${offsetY} ${blur} ${spread} ${color}`,
+})
+
+/**
+ * Transform an array of cubic bezier timing values to a css cubic-bezier() call
+ * @see https://design-tokens.github.io/community-group/format/#cubic-bezier
+ */
+StyleDictionary.registerTransform({
+  type: 'value',
+  name: 'cubic-bezier/css',
+  matcher: token => token.type === 'cubicBezier',
+  transformer: ({ value }) =>
+    `cubic-bezier(${value.join(', ')})`,
+})
+
+const CSS_TRANSFORMS = [
+  'shadow/css',
+  'cubic-bezier/css',
+  'attribute/cti',
+  'name/cti/kebab',
+  'time/seconds',
+  'content/icon',
+  'size/rem',
+  'color/css',
+];
+
+StyleDictionary.registerTransformGroup({ name: 'css', transforms: CSS_TRANSFORMS })
+StyleDictionary.registerTransformGroup({ name: 'scss', transforms: CSS_TRANSFORMS })
+
+/**
+ * Lit CSS object
+ * @example ```js
+ * import { resetStyles } from '@rhds/tokens/reset.css.js';
+ *
+ * class RhJazzHands extends LitElement {
+ *   // reset all RHDS variables to defaults
+ *   static styles = [resetStyles];
+ * }
+ * ```
+ */
+StyleDictionary.registerFormat({
+  name: 'css/lit',
+  formatter: ({ file, dictionary, options }) => `${fileHeader({file})}
+import { css } from 'lit';
+export const resetStyles = css\`
+:host {
+${formattedVariables({ format: 'css', dictionary, outputReferences: options.outputReferences })}
+}\`;
+export default resetStyles;`,
+});
+
+/**
+ * Exports VSCode style snippets for editor support
+ */
+StyleDictionary.registerFormat({
+  name: 'snippets/vscode',
+  formatter: ({ dictionary }) =>
+    JSON.stringify(Object.fromEntries(
+      dictionary.allProperties.map(prop => [
+        prop.title ?? prop.name,
+        {
+          scope: 'css,scss',
+          prefix: [`--${prop.name}`],
+          body: [`var(--${prop.name}, ${prop.value})`],
```

Comment:
I'd like higher-level tokens to use the whole variable stack here.

e.g. if currently, given the prefix `--rh-theme-color-link`, the user gets:
```css
color: var(--rh-theme-color-link, #0066cc);
```
Instead, I'd like them to get:
```css
color: var(--rh-theme-color-link, var(--rh-color-interactive-blue, var(--rh-color-blue-500, #0066cc));

Reply:
I think the latter example is probably sufficient.

```color: var(--rh-theme-color-link, var(--rh-color-interactive-blue, #0066cc);```